scripts/train.py

```diff
@@ -31,25 +31,25 @@ def setup_config_and_paths(args):
     for local_path in ['embedding_model_location','MSA_data_folder','MSA_weight_data_folder','path_to_hhfilter']:
         if getattr(args, local_path) and local_path in args_setup_from_config:
             setattr(args, local_path, args.data_location + os.sep + getattr(args, local_path))
-    if not os.path.exists(args.data_location + os.sep + 'model_predictions'): os.mkdir(args.data_location + os.sep + 'model_predictions')
-    if not os.path.exists(args.data_location + os.sep + 'checkpoint'): os.mkdir(args.data_location + os.sep + 'checkpoint')
+    if not os.path.exists(args.data_location + os.sep + 'model_predictions'): os.makedirs(args.data_location + os.sep + 'model_predictions', exist_ok=True)
+    if not os.path.exists(args.data_location + os.sep + 'checkpoint'): os.makedirs(args.data_location + os.sep + 'checkpoint', exist_ok=True)
     args.output_scores_location = args.data_location + os.sep + 'model_predictions' + os.sep + args.model_name_suffix
-    if not os.path.exists(args.output_scores_location): os.mkdir(args.output_scores_location)
+    if not os.path.exists(args.output_scores_location): os.makedirs(args.output_scores_location, exist_ok=True)
     args.model_location = args.data_location + os.sep + 'checkpoint' + os.sep + args.model_name_suffix
-    if not os.path.exists(args.model_location): os.mkdir(args.model_location)
+    if not os.path.exists(args.model_location): os.makedirs(args.model_location, exist_ok=True)
     if args.assay_data_location and not args.assay_data_folder: args.assay_data_folder = [ os.sep.join(args.assay_data_location.split(os.sep)[:-1]) ] # args.assay_data_folder is a list
 
     # Target config
     args.target_config=json.load(open(args.target_config_location))
     zero_shot_predictions_mapping={
-            "MSA_Transformer_pred": "MSA_Transformer_ensemble",
-            "ESM1v_pred": "ESM1v_ensemble",
-            "ESM2_15B_pred": "ESM2_15B",
-            "ESM2_3B_pred": "ESM2_3B",
-            "ESM2_650M_pred": "ESM2_650M",
-            "TranceptEVE_pred": "TranceptEVE_L",
-            "Tranception_pred": "Tranception_L",
-            "DeepSequence_pred": "DeepSequence_ensemble"
+            "MSA_Transformer_pred": "esm_msa1b_t12_100M_UR50S_ensemble",
+            "ESM1v_pred": "Ensemble_ESM1v",
+            "ESM2_15B_pred": "esm2_t48_15B_UR50D",
+            "ESM2_3B_pred": "esm2_t36_3B_UR50D",
+            "ESM2_650M_pred": "esm2_t33_650M_UR50D",
+            "TranceptEVE_pred": "avg_score",
+            "Tranception_pred": "avg_score",
+            "DeepSequence_pred": "evol_indices_ensemble"
```

Comment:
Let's revert this change -- the prior names are the expected ones in the merged files / ProteinNPT data package.

Reply:
They're now reverted!